frontend/src/components/Menu.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+
+function Menu() {
+  const [isMenu, setIsMenu] = useState(true);
+  const handleChangeMenu = () => {
+    setIsMenu(!isMenu);
+    console.info(isMenu);
+  };
+
+  return (
+    <div className="firstMenu">
+      <p>Quelle est votre pseudo ?</p>
+      <input type="text" className="inputFirstMenu" />
+      <button
+        type="button"
+        onClick={handleChangeMenu}
+        className="buttonFirstMenu"
+      >
+        Jouer
+      </button>
+    </div>
```

Comment:
a container with an input and a validation button: looks more like a form (and so the p should be a label)